docs/2.10.0/docs/query/pqs-user-guide.rst

```diff
@@ -2985,13 +3026,403 @@ Based on the metrics described above, it is possible to build a comprehensive da
+   Kind           : Internal
+   Start time     : 2024-11-06 03:16:43.021507 +0000 UTC
+   End time       : 2024-11-06 03:16:43.024872 +0000 UTC
+   Status code    : Unset
+   Status message :
+   Attributes:
+        -> scribe.watermark.offset: Str(000000000000000261)
+        -> scribe.watermark.ix: Int(384)
+   Links:
+   SpanLink #0
+        -> Trace ID: 76c58361d46c08761c37ef5821e8fb78
+        -> ID: 6051f05f10af0399
+        -> TraceState:
+        -> Attributes::
+             -> target: Str(↥ persist to datastore)
+
+   Span #116
+   Trace ID       : 71e67e2420deeef36ef3efacea6399dc
+   Parent ID      : 161b5911e7a0ec18
+   ID             : 33ab3918ebfe138d
+   Name           : acquire connection
+   Kind           : Internal
+   Start time     : 2024-11-06 03:16:43.022009 +0000 UTC
+   End time       : 2024-11-06 03:16:43.022222 +0000 UTC
+   Status code    : Unset
+   Status message :
+
+   Span #6
+   Trace ID       : 71e67e2420deeef36ef3efacea6399dc
+   Parent ID      : 161b5911e7a0ec18
+   ID             : 1a66240dfd597654
+   Name           : UPDATE scribe.__watermark
+   Kind           : Client
+   Start time     : 2024-11-06 03:16:43.022737084 +0000 UTC
+   End time       : 2024-11-06 03:16:43.023134917 +0000 UTC
+   Status code    : Unset
+   Status message :
+   Attributes:
+        -> db.operation: Str(UPDATE)
+        -> db.sql.table: Str(__watermark)
+        -> db.name: Str(scribe)
+        -> db.connection_string: Str(postgresql://postgres-scribe:5432)
+        -> server.address: Str(postgres-scribe)
+        -> server.port: Int(5432)
+        -> db.user: Str(pguser)
+        -> db.statement: Str(update __watermark set "offset" = ?, ix = ?;)
+        -> db.system: Str(postgresql)
+
+   Span #117
+   Trace ID       : 71e67e2420deeef36ef3efacea6399dc
+   Parent ID      : 161b5911e7a0ec18
+   ID             : f0b24fb074fe41f8
+   Name           : commit transaction
+   Kind           : Internal
+   Start time     : 2024-11-06 03:16:43.023629 +0000 UTC
+   End time       : 2024-11-06 03:16:43.024157 +0000 UTC
+   Status code    : Unset
+   Status message :
+
+.. raw:: html
+
+   </details>
+
+Trace Context Propagation
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PQS is an intermediary between a ledger instance and downstream applications that would prefer to access data through SQL rather than in streaming manner from Ledger API directly. Despite forming a pipeline between two data storage systems (Canton and Postgres), PQS stores the `original ledger transaction's trace context <https://docs.daml.com/app-dev/bindings-java/open-tracing.html#continue-spans-across-different-applications>`__ for the purposes of propagation rather than its own. This allows downstream applications to decide for themselves how they want to connect to the original submission's trace (as a child span or as a new trace connected through `span links <https://opentelemetry.io/docs/specs/otel/overview/#links-between-spans>`__).
```

Comment:
```suggestion
PQS is an intermediary between a ledger instance and downstream applications that access data through SQL rather than by streaming directly from the Ledger API. Despite forming a pipeline between two data storage systems (Canton and Postgres), PQS stores the `original ledger transaction's trace context <https://docs.daml.com/app-dev/bindings-java/open-tracing.html#continue-spans-across-different-applications>`__ for propagation. This allows downstream applications to decide for themselves how they want to connect to the original submission's trace (as a child span or as a new trace connected through `span links <https://opentelemetry.io/docs/specs/otel/overview/#links-between-spans>`__).
```